data/json/mapgen/musicstore.json

```diff
@@ -31,7 +31,7 @@
             "collection": [
               { "item": "case_violin" },
               { "item": "violin_golden" }
-            ], "prob": 15
+            ], "prob": 50
```

Comment:
What's the rationale? It supposed to be a rare find.

Reply:
Likely because the weight of music stores themselves adds an additional layer to its generation.

Reply:
>Likely because the weight of music stores themselves adds an additional layer to its generation.

This is the rationale.

Reply:
That said, dunno if 50% is enough or too much. Seems okay but others might suggest a different rate. I'd assume this is also to the benefit of Arcana mod?